library/src/main/java/com/novoda/downloadmanager/Migration.java

```diff
@@ -5,14 +5,21 @@
 
 class Migration {
 
+    enum Type {
```

Comment:
This is used to drive whether we should retrieve the file size for a given download. For partials we don't care, it should be zero so that the `DownloadManager` can request from the network again.